js/apps/system/_admin/aardvark/APP/frontend/js/models/arangoCollectionModel.js

```diff
@@ -35,7 +35,7 @@
       $.ajax({
         type: 'GET',
         cache: false,
-        url: arangoHelper.databaseUrl('/_api/collection/' + this.get('id') + '/shards?details=true'),
+        url: arangoHelper.databaseUrl('/_api/collection/' + encodeURIComponent(this.get('id')) + '/shards?details=true'),
```

Comment:
Just some note: You could use here this.escape('id'), see Backbone Documentation:
- https://backbonejs.org/#Model-escape

```
Similar to get, but returns the HTML-escaped version of a model's attribute. If you're interpolating data from the model into HTML, using escape to retrieve attributes will prevent XSS attacks.
```

Reply:
But feel free to ignore. This will do the same thing. Your choice.

Reply:
We are not HTML-escaping here, but URL-escaping. This is slightly different.